src/repositories/books.repository.js

```diff
@@ -0,0 +1,88 @@
+const db = require("../../db");
+
+const getAllBooks = async () => {
+  const sqlQuery = `SELECT * FROM books`;
+
+  try {
+    const result = await db.query(sqlQuery);
+    return result.rows;
+  } catch (error) {
+    console.error(error);
```

Comment:
Always a good idea to give yourself more context with errors like this. Something as simple as:
```js
console.error("error getting all books", error)
```

In practice, you'll likely not see an error from this handler for months, then you will see a random error in your logs and not know where that error is being throw.